<launch>

  <!-- Bring up Kinect -->
  <include file="$(find openni_launch)/launch/openni.launch">
    <!-- Set cam info URLs, if not in the default location -->
    <arg name="depth_camera_info_url" default="package://robot_connect/calibrations/depth_0000000000000000.yaml" />
  </include>


  <include file="$(find robot_connect)/launch/open_webcam.launch"/>

  <!-- BRING UP EXTERNAL CAMERA HERE -->

  <!-- In Fuerte, you can instead load image_proc as nodelets into the
       OpenNI nodelet manager: -->
 <include ns="logitech_cam" file="$(find image_proc)/launch/image_proc.launch">
    <arg name="manager" value="/camera_nodelet_manager" />
  </include>

  <!-- Extrinsic transform publisher -->
  <include file="$(find camera_pose_calibration)/blocks/calibration_tf_publisher.launch">
    <arg name="cache_file" value="$(find robot_connect)/calibrations/kinect_extrinsics_cache.bag" />
  </include>

  <node name="contrast_augmenter" pkg="contrast" type="contrast_augmenter" args="image:=/camera/ir/image_raw"/>

  <!-- Register depth to external RGB camera and create point clouds -->
  <!-- Replace 'my_camera' with the correct namespace -->
<!--
  <include file="$(find openni_launch)/launch/includes/depth_registered.launch">
    <arg name="manager" value="/camera_nodelet_manager" />
    <arg name="rgb" value="logitech_cam" />
    <arg name="depth" value="camera/depth" />

    <arg name="depth_registered" value="logitech_cam_depth" />
    <arg name="suffix" value="logitech_cam_depth" />
  </include>
-->

</launch>
